resources/EPvalidator.rb

```diff
@@ -364,11 +364,6 @@ def self.run_validator(hpxml_doc)
         "Ducts[DuctType='return']" => one_or_more, # See [HVACDuct]
       },
 
-      ## [HVACDistType=DSE]
-      "/HPXML/Building/BuildingDetails/Systems/HVAC/HVACDistribution[DistributionSystemType[Other='DSE']]" => {
-        "[AnnualHeatingDistributionSystemEfficiency | AnnualCoolingDistributionSystemEfficiency]" => one_or_more,
```

Comment:
We might still want EPvalidator to be open for DSE option, so maybe still need this?